Release a patch version 1.3.4 to remove use of syreclabs.com/go/faker

Hi @AlekSi,

Would it be possible to create a tag v1.3.4 to fix issues with syreclabs.com/go/faker?
I see that the library has been replaced by github.com/brianvoe/gofakeit with the PR https://github.com/go-reform/reform/pull/195 but that hasn't been released yet, thus isn't available through `go get -u gpkg.in/reform.v1`.

For references, issues with syreclabs.com/go/faker:
- https://github.com/go-reform/reform/issues/194
- https://github.com/dmgk/faker/issues/14

```
$ go mod download
#... more fetches
go: finding gopkg.in/reform.v1 v1.3.3
go: syreclabs.com/go/faker@v1.2.0: unrecognized import path "syreclabs.com/go/faker" (https fetch: Get https://syreclabs.com/go/faker?go-get=1: x509: certificate signed by unknown authority)
# ... more fetches
go: error loading module requirements
##[error]Process completed with exit code 1.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,9 +3,11 @@ sudo: required
 language: go
 
 go:
-  - 1.8.x
-  - 1.9.x
   - 1.10.x
+  - 1.11.x
+  - 1.12.x
+  - 1.13.x
+  - 1.14.x
   - master
 
 go_import_path: gopkg.in/reform.v1

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,7 +1,8 @@
 # Changelog
 
-## v1.3.4 (2020-06-XX, https://github.com/go-reform/reform/milestones/v1.3.4)
+## v1.3.4 (2020-06-25, https://github.com/go-reform/reform/milestones/v1.3.4)
 
+* Make reform generator work with Go 1.15.
 * Replace syreclabs.com/go/faker with github.com/brianvoe/gofakeit.
 
 ## v1.3.3 (2018-12-11, https://github.com/go-reform/reform/milestones/v1.3.3)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.3.4 (2020-06-XX, https://github.com/go-reform/reform/milestones/v1.3.4)
+
+* Replace syreclabs.com/go/faker with github.com/brianvoe/gofakeit.
+
 ## v1.3.3 (2018-12-11, https://github.com/go-reform/reform/milestones/v1.3.3)
 
 * Fix tests for Go 1.12.

--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ deps:
 
 	go get -u github.com/AlekSi/pointer
 	go get -u github.com/stretchr/testify/...
-	go get -u syreclabs.com/go/faker
+	go get -u github.com/brianvoe/gofakeit
 	go get -u gopkg.in/alecthomas/gometalinter.v1
 	go get -u github.com/AlekSi/gocoverutil
 

--- a/db_test.go
+++ b/db_test.go
@@ -4,7 +4,7 @@ import (
 	"errors"
 
 	"github.com/AlekSi/pointer"
-	"syreclabs.com/go/faker"
+	"github.com/brianvoe/gofakeit"
 
 	"gopkg.in/reform.v1"
 	"gopkg.in/reform.v1/dialects/postgresql"
@@ -15,7 +15,7 @@ func (s *ReformSuite) TestBeginCommit() {
 	s.Require().NoError(s.tx.Rollback())
 	s.q = nil
 
-	person := &Person{ID: 42, Email: pointer.ToString(faker.Internet().Email())}
+	person := &Person{ID: 42, Email: pointer.ToString(gofakeit.Email())}
 
 	tx, err := DB.Begin()
 	s.Require().NoError(err)
@@ -31,7 +31,7 @@ func (s *ReformSuite) TestBeginRollback() {
 	s.Require().NoError(s.tx.Rollback())
 	s.q = nil
 
-	person := &Person{ID: 42, Email: pointer.ToString(faker.Internet().Email())}
+	person := &Person{ID: 42, Email: pointer.ToString(gofakeit.Email())}
 
 	tx, err := DB.Begin()
 	s.Require().NoError(err)
@@ -51,8 +51,8 @@ func (s *ReformSuite) TestErrorInTransaction() {
 	s.Require().NoError(s.tx.Rollback())
 	s.q = nil
 
-	person1 := &Person{ID: 42, Email: pointer.ToString(faker.Internet().Email())}
-	person2 := &Person{ID: 43, Email: pointer.ToString(faker.Internet().Email())}
+	person1 := &Person{ID: 42, Email: pointer.ToString(gofakeit.Email())}
+	person2 := &Person{ID: 43, Email: pointer.ToString(gofakeit.Email())}
 
 	// commit works
 	tx, err := DB.Begin()
@@ -91,8 +91,8 @@ func (s *ReformSuite) TestAbortedTransaction() {
 	s.Require().NoError(s.tx.Rollback())
 	s.q = nil
 
-	person1 := &Person{ID: 42, Email: pointer.ToString(faker.Internet().Email())}
-	person2 := &Person{ID: 43, Email: pointer.ToString(faker.Internet().Email())}
+	person1 := &Person{ID: 42, Email: pointer.ToString(gofakeit.Email())}
+	person2 := &Person{ID: 43, Email: pointer.ToString(gofakeit.Email())}
 
 	// commit fails
 	tx, err := DB.Begin()
@@ -122,7 +122,7 @@ func (s *ReformSuite) TestInTransaction() {
 	s.Require().NoError(s.tx.Rollback())
 	s.q = nil
 
-	person := &Person{ID: 42, Email: pointer.ToString(faker.Internet().Email())}
+	person := &Person{ID: 42, Email: pointer.ToString(gofakeit.Email())}
 
 	// error in closure
 	err := DB.InTransaction(func(tx *reform.TX) error {

--- a/querier_commands_test.go
+++ b/querier_commands_test.go
@@ -6,7 +6,7 @@ import (
 	"time"
 
 	"github.com/AlekSi/pointer"
-	"syreclabs.com/go/faker"
+	"github.com/brianvoe/gofakeit"
 
 	"gopkg.in/reform.v1"
 	"gopkg.in/reform.v1/dialects/postgresql"
@@ -14,7 +14,7 @@ import (
 )
 
 func (s *ReformSuite) TestInsert() {
-	newEmail := faker.Internet().Email()
+	newEmail := gofakeit.Email()
 	person := &Person{Email: &newEmail}
 	err := s.q.Insert(person)
 	s.NoError(err)
@@ -34,7 +34,7 @@ func (s *ReformSuite) TestInsert() {
 
 func (s *ReformSuite) TestInsertWithValues() {
 	t := time.Now()
-	newEmail := faker.Internet().Email()
+	newEmail := gofakeit.Email()
 	person := &Person{Email: &newEmail, CreatedAt: t, UpdatedAt: &t}
 	err := s.q.Insert(person)
 	s.NoError(err)
@@ -53,7 +53,7 @@ func (s *ReformSuite) TestInsertWithValues() {
 }
 
 func (s *ReformSuite) TestInsertWithPrimaryKey() {
-	newEmail := faker.Internet().Email()
+	newEmail := gofakeit.Email()
 	person := &Person{ID: 50, Email: &newEmail}
 	err := insertPersonWithID(s.T(), s.q, person)
 	s.NoError(err)
@@ -103,7 +103,7 @@ func (s *ReformSuite) TestInsertIntoView() {
 
 func (s *ReformSuite) TestInsertColumns() {
 	t := time.Now()
-	newEmail := faker.Internet().Email()
+	newEmail := gofakeit.Email()
 	person := &Person{Email: &newEmail, CreatedAt: t, UpdatedAt: &t}
 	columns := []string{"name", "email", "created_at", "updated_at"}
 	err := s.q.InsertColumns(person, columns...)
@@ -141,8 +141,8 @@ func (s *ReformSuite) TestInsertColumnsIntoView() {
 }
 
 func (s *ReformSuite) TestInsertMulti() {
-	newEmail := faker.Internet().Email()
-	newName := faker.Name().Name()
+	newEmail := gofakeit.Email()
+	newName := gofakeit.Name()
 	person1, person2 := &Person{Email: &newEmail}, &Person{Name: newName}
 	err := s.q.InsertMulti(person1, person2)
 	s.NoError(err)
@@ -161,8 +161,8 @@ func (s *ReformSuite) TestInsertMulti() {
 }
 
 func (s *ReformSuite) TestInsertMultiWithPrimaryKeys() {
-	newEmail := faker.Internet().Email()
-	newName := faker.Name().Name()
+	newEmail := gofakeit.Email()
+	newName := gofakeit.Name()
 	person1, person2 := &Person{ID: 50, Email: &newEmail}, &Person{ID: 51, Name: newName}
 
 	withIdentityInsert(s.T(), s.q, "people", func() {
@@ -221,7 +221,7 @@ func (s *ReformSuite) TestUpdate() {
 	err = s.q.FindByPrimaryKeyTo(&person, 102)
 	s.NoError(err)
 
-	person.Email = pointer.ToString(faker.Internet().Email())
+	person.Email = pointer.ToString(gofakeit.Email())
 	err = s.q.Update(&person)
 	s.NoError(err)
 	s.Equal(personCreated, person.CreatedAt)
@@ -234,7 +234,7 @@ func (s *ReformSuite) TestUpdate() {
 }
 
 func (s *ReformSuite) TestUpdateOverwrite() {
-	newEmail := faker.Internet().Email()
+	newEmail := gofakeit.Email()
 	person := Person{ID: 102, Email: pointer.ToString(newEmail)}
 	err := s.q.Update(&person)
 	s.NoError(err)
@@ -250,8 +250,8 @@ func (s *ReformSuite) TestUpdateOverwrite() {
 }
 
 func (s *ReformSuite) TestUpdateColumns() {
-	newName := faker.Name().Name()
-	newEmail := faker.Internet().Email()
+	newName := gofakeit.Name()
+	newEmail := gofakeit.Email()
 
 	for p, columns := range map[*Person][]string{
 		&Person{Name: "Elfrieda Abbott", Email: &newEmail}:                             {"email", "updated_at"},
@@ -294,8 +294,8 @@ func (s *ReformSuite) TestUpdateColumns() {
 }
 
 func (s *ReformSuite) TestUpdateView() {
-	newName := faker.Name().Name()
-	newEmail := faker.Internet().Email()
+	newName := gofakeit.Name()
+	newEmail := gofakeit.Email()
 
 	for p, columns := range map[*Person][]string{
 		&Person{Name: "Elfrieda Abbott", Email: &newEmail}:                             {"email", "updated_at"},
@@ -340,7 +340,7 @@ func (s *ReformSuite) TestUpdateView() {
 }
 
 func (s *ReformSuite) TestSave() {
-	newName := faker.Name().Name()
+	newName := gofakeit.Name()
 	person := &Person{Name: newName}
 	err := s.q.Save(person)
 	s.NoError(err)
@@ -351,7 +351,7 @@ func (s *ReformSuite) TestSave() {
 	s.Nil(person2.(*Person).Email)
 	s.Equal(person, person2)
 
-	newEmail := faker.Internet().Email()
+	newEmail := gofakeit.Email()
 	person.Email = &newEmail
 	err = s.q.Save(person)
 	s.NoError(err)
@@ -364,7 +364,7 @@ func (s *ReformSuite) TestSave() {
 }
 
 func (s *ReformSuite) TestSaveWithPrimaryKey() {
-	newName := faker.Name().Name()
+	newName := gofakeit.Name()
 	person := &Person{ID: 99, Name: newName}
 
 	withIdentityInsert(s.T(), s.q, "people", func() {
@@ -427,7 +427,7 @@ func (s *ReformSuite) TestCommandsSchema() {
 		s.T().Skip("only PostgreSQL supports schemas")
 	}
 
-	legacyPerson := &LegacyPerson{Name: pointer.ToString(faker.Name().Name())}
+	legacyPerson := &LegacyPerson{Name: pointer.ToString(gofakeit.Name())}
 	err := s.q.Save(legacyPerson)
 	s.NoError(err)
 	err = s.q.Save(legacyPerson)

--- a/querier_examples_test.go
+++ b/querier_examples_test.go
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/AlekSi/pointer"
-	"syreclabs.com/go/faker"
+	"github.com/brianvoe/gofakeit"
 
 	"gopkg.in/reform.v1"
 	"gopkg.in/reform.v1/dialects/postgresql"
@@ -123,20 +123,20 @@ var persons = []reform.Struct{
 		Email: pointer.ToString("alexey.palazhchenko@gmail.com"),
 	},
 	&Person{
-		Name:  faker.Name().Name(),
-		Email: pointer.ToString(faker.Internet().Email()),
+		Name:  gofakeit.Name(),
+		Email: pointer.ToString(gofakeit.Email()),
 	},
 	&Person{
-		Name:  faker.Name().Name(),
-		Email: pointer.ToString(faker.Internet().Email()),
+		Name:  gofakeit.Name(),
+		Email: pointer.ToString(gofakeit.Email()),
 	},
 	&Person{
-		Name:  faker.Name().Name(),
-		Email: pointer.ToString(faker.Internet().Email()),
+		Name:  gofakeit.Name(),
+		Email: pointer.ToString(gofakeit.Email()),
 	},
 	&Person{
-		Name:  faker.Name().Name(),
-		Email: pointer.ToString(faker.Internet().Email()),
+		Name:  gofakeit.Name(),
+		Email: pointer.ToString(gofakeit.Email()),
 	},
 }
 

